cfgmgr/buffermgrdyn.cpp

```diff
@@ -1462,6 +1462,26 @@ task_process_status BufferMgrDynamic::refreshPgsForPort(const string &port, cons
                 continue;
             }
 
+            // If cable len is 0m, remove lossless PG, keep lossy PG.
+            if (cable_length == "0m" && portPg.lossless)
+            {
+                if (oldProfile.empty())
+                {
+                    SWSS_LOG_NOTICE("No lossless profile found for port %s when cable length is set to '0m'.", port.c_str());
```

Comment:
It isn't worth a notice message
```suggestion
                    SWSS_LOG_INFO("No lossless profile found for port %s when cable length is set to '0m'.", port.c_str());
```